docs/tutorials/own_machine/06_customize_env.rst

```diff
@@ -0,0 +1,82 @@
+.. _customize_env:
+
+Customize your Renku Project Environment
```

Comment:
I believe there is a how-to for this already?

Reply:
Adding packages is certainly documented in the first tutorial (`first_steps/04_add_packages.rst`). However my goal is to have this tutorial be standalone, where the user doesn't have to do the first tutorial as a prerequisite (my vision is that these two tutorials address two distinct user groups). 

Perhaps I could convert "installing packages" to a sort of [module](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/reuse.html#include-a-shared-file) and reuse it in both tutorials? However one has screenshots from RenkuLab and the other doesn't... 🤷‍♀️ 

There is also a little info in "Customizing sessions" (the same how-to that I link to for more details on Docker), but it's not very detailed: 

> The most basic modifications are installations of additional packages. This can be done automatically for Python and R projects if you add the packages you want to ``requirements.txt`` and ``install.R`` respectively.

Thoughts?